app/controllers/team/projects_controller.rb

```diff
@@ -17,4 +17,11 @@ def unassigned
 
     @pager, @projects = pagy_array(ByTeamProjectFetcherService.new(current_user.team).unassigned)
   end
+
+  def handed_over
+    authorize Project, :handed_over?
+
+    @current_users_team = current_user.team
+    @pager, @projects = pagy_array(ByRegionProjectFetcherService.new.regional_casework_services_projects(@current_users_team))
```

Comment:
Now that it is available, could you call the new `AcademiesApiPreFetcherService` here to make this view feel responsive:

`AcademiesApiPreFetcherService.new.call!(@projects)`

This is the pattern we use now anytime we have a paged view of projects.